src/draw/lv_draw_image.h

```diff
@@ -71,7 +71,7 @@ struct _lv_draw_image_dsc_t {
     /**Describes how to blend the pixels of the image to the background.
      * See `lv_blend_mode_t` for more details.
      */
-    lv_blend_mode_t blend_mode : 3;
+    uint16_t blend_mode : 4;
```

Comment:
Ah, sorry I meant this:
```suggestion
    lv_blend_mode_t blend_mode : 4;
```
to keep the type. 😓

Reply:
But it will still be the issue again when lv_blend_mode_t adds more enum items in the future. So, maybe we should have a way to let other guys to know that.

Reply:
Note: Only C++11/C23 or later can limit the type of enum constants.
https://en.cppreference.com/w/c/language/enum

Reply:
I followed your suggestion. But I hope there is some way to warn someone who wants to modify that.